Modules/Loadable/Markups/MRMLDM/vtkMRMLMarkupsDisplayableManagerHelper.cxx

```diff
@@ -327,18 +328,30 @@ void vtkMRMLMarkupsDisplayableManagerHelper::AddInteractionWidget(vtkMRMLMarkups
   // There should not be a widget for the new node
   if (this->GetInteractionWidget(markupsDisplayNode) != nullptr)
   {
-    vtkErrorMacro("vtkMRMLMarkupsDisplayableManagerHelper: A widget is already associated to this node");
+    vtkErrorMacro("vtkMRMLMarkupsDisplayableManagerHelper: An interaction widget is already associated to this node");
+    return;
+  }
+
+  vtkMRMLNode* markupsNode = markupsDisplayNode->GetDisplayableNode();
+  if (!markupsNode)
+  {
+    // No markups node associated with the display node.
+    // Nothing to display.
```

Comment:
Add something like you described in the commit message: "This is not an error, the widget will be created once both nodes are present"